packages/fast-form-generator-react/src/form/form.tsx

```diff
@@ -230,12 +233,19 @@ class Form extends React.Component<IFormProps & IManagedClasses<IFormClassNameCo
      * Generate the section to be shown
      */
     private generateSection(): JSX.Element {
+        const lastComponentTrackerIndex: number = this.state.componentTracker.length - 1;
+        const schema: any = isRootLocation(this.state.activeSchemaLocation)
+            ? this.state.componentTracker[lastComponentTrackerIndex].schema
```

Comment:
You should store `this.state.componentTracker[lastComponentTrackerIndex].schema` in a variable so you don't access it more than once.